src/main/java/Duke.java

```diff
@@ -1,39 +1,73 @@
 import java.util.Scanner;
+import java.util.Arrays;
 
 public class Duke {
     public static void main(String[] args) {
         Task[] tasks = new Task[100];
-        int count = 1;
+        int count = 0;
+        boolean isBye = false;
         System.out.println("    ____________________________________________________________");
         System.out.println("    Hello! I'm Duke");
         System.out.println("    What can I do for you?");
         System.out.println("    ____________________________________________________________");
         Scanner sc = new Scanner(System.in);
-        String string = sc.nextLine();
-        while (!string.equals("bye")) {
+        while (!isBye) {
+            String description;
+            String date;
+            System.out.println();
+            String string = sc.nextLine();
             System.out.println("    ____________________________________________________________");
             String[] stringSplit = string.split(" ");
-            if (string.equals("list")) {
+            switch (stringSplit[0]) {
```

Comment:
Use switch to deal with different types of commands is more clear than my method. Nice!